cockatrice/src/window_main.cpp

```diff
@@ -393,9 +393,9 @@ QString MainWindow::extractInvalidUsernameMessage(QString & in)
 {
     QString out = tr("Invalid username.") + "<br/>";
     QStringList rules = in.split(QChar('|'));
-    if (rules.size() == 7)
+    if (rules.size() == 9)
```

Comment:
It would be nice to make this more backwards compatible, so that when a client that includes this change connects to a server it would be able to handle both the old `rules.size() == 7` (old server not including this change) and the new `rules.size() == 9` (new server including this change).
You could just use to nested `if`s or a switch with fallthrough.